events/event.h

```diff
@@ -42,7 +42,10 @@ namespace QMatrixClient
                 RoomName = RoomStateEventBase + 1,
                 RoomAliases, RoomCanonicalAlias, RoomMember, RoomTopic,
                 RoomAvatar, RoomEncryption,
-                CallInvite, CallCandidates, CallAnswer, CallHangup,
+                CallInvite = 0x2100,
+                CallCandidates = CallInvite + 1,
+                CallAnswer = CallCandidates + 1,
+                CallHangup = CallAnswer + 1,
```

Comment:
If you use `0x2100` for calls, push `Type::Reserved` to `0x4000` or something like that. Or instead, something like `0x1100` would fit much better (because `0x2100` does not have the `0x1000` "room event" bit - we don't have any trouble with it right now because we don't have `Event::isRoomEvent()` but who knows what will come).